cradlepoint/pages/api/getEngagement.js

```diff
@@ -0,0 +1,37 @@
+import connectToDb from "../../util/mongodb";
+const { ObjectId } = require('mongodb');
+
+/*
+ gets all the active engagements from the database
+*/
+export default async (req, res) => {
+  if ( ObjectId.isValid(req.query._id) ) {
```

Comment:
You changed the query param style in each of your changed API's from "objectId" to "_id". While I agree that this is probably a good change, I would have liked to have seen a comment about it. Also a comment for testing this should have been left, because I assumed it was tested he same as before.

Reply:
This is actually not different. This one is pulling the actual engagement. The other changes are instances where the id isn't what we are actually pulling. ie pulling test plans, but doing so by using an engagement id and not a test plan id. The other two endpoints are both pulling data for a test plan using a test plan id

Reply:
It is different, and you didn't mark the change from "req.query.testPlanId" to "req.query._id". This is what i meant by "object_id", sorry if that was confusing, but there is a difference in the File Changes.

I mostly just wanted this change to have been noted in a comment.